### ISSUE TITLE
[BugFix] [RHEL/6] One instance of "xccdf_org.ssgproject.content_rule_security_patches_up_to_date - If a <xccdf:check-content-ref> is in a security patches up-to-date rule then the @name SHALL be omitted from the <xccdf:check-content-ref>. This requirement was removed in SCAP Schematron version 1.1. " error

### ISSUE THREAD

**Issue by iankko** [OpenSCAP/scap-security-guide#1091]
Performing a current RHEL-6 SSG Benchmark validation with the SCAP NIST content suite on 2016-03-18 (https://jlieskov.fedorapeople.org/2016-03-18-scap-validation-result.html) returns one instance of the following error:
```
171-1 	1(of 1)
ERROR 	SCHEMATRON

xccdf_org.ssgproject.content_rule_security_patches_up_to_date -
If a <xccdf:check-content-ref> is in a security patches up-to-date rule then
the @name SHALL be omitted from the <xccdf:check-content-ref>.
This requirement was removed in SCAP Schematron version 1.1.

/*:data-stream-
collection[namespace-uri()='http://scap.nist.gov/schema/scap/source/1.2']
[1]/*:component[namespace-uri()='http://scap.nist.gov/schema/scap/source/1.2']
[2]/*:Benchmark[namespace-uri()='http://checklists.nist.gov/xccdf/1.2'][1]/*:Group[namespace-
uri()='http://checklists.nist.gov/xccdf/1.2'][2]/*:Group[namespace-uri()='http://checklists.nist.gov/xccdf
/1.2'][2]/*:Group[namespace-uri()='http://checklists.nist.gov/xccdf/1.2'][2]/*:Rule[namespace-
uri()='http://checklists.nist.gov/xccdf/1.2'][4]/*:check[namespace-uri()='http://checklists.nist.gov/xccdf
/1.2'][2]/*:check-content-ref[namespace-uri()='http://checklists.nist.gov/xccdf/1.2'][1] 	if( 
exists(ancestor::xccdf:Rule[1][starts-with(@id,'xccdf_') and 
ends-with(@id,'_rule_security_patches_up_to_date')]) ) then not(exists(@name)) else true() 
```

**Comment by iankko** [comment]
Moving back to the pool so others can have a look too.

**Comment by iankko** [comment]
The ```scap-security-guide-nist-testsuite``` jenkins job has been modified to whitelist the specific error message for this problem (see next comment for explanation why):

```
ERROR: SCHEMATRON - [ssg-rhel6-ds.xml] 
xccdf_org.ssgproject.content_rule_security_patches_up_to_date -
If a <xccdf:check-content-ref> is in a security patches up-to-date
rule then the @name SHALL be omitted from the <xccdf:check-
content-ref>. This requirement was removed in SCAP Schematron version 1.1.
```

**Comment by iankko** [comment]
The reason for this issue (rather than to be a problem in the SSG content itself) has been recognized to be a deficiency in the SCAP NIST content validation itself. As of ```scapval-1.2.1.13.jar``` version it is not able to recognize remotely referenced OVAL check being present in the benchmark, and issuing ```ERROR``` in this case. This behaviour has been confirmed by NIST as not the desired one and as such will change in future ScapVal versions.

Instead of issuing an ```ERROR``` future version will issue a ```WARNING```. Future ScapVal version should be made available by start of April 2016.

For now we have whitelisted this error in the testsuite Jenkins job, and as such no further action is necessary from our end.

Closing as resolved.
